conn.go

```diff
@@ -604,9 +604,8 @@ func (c *Client) pingLoop(ctx context.Context, errs chan error, wg *sync.WaitGro
 			}
 
 			c.conn.mu.RLock()
-			if pingSent && time.Since(c.conn.lastPong) > c.Config.PingDelay+(60*time.Second) {
-				// It's 60 seconds over what out ping delay is, connection
-				// has probably dropped.
+			if pingSent && time.Since(c.conn.lastPong) > c.Config.PingTimeout {
```

Comment:
This won't actually work the way you might intend it to. `c.conn.lastPong` is the last successful `PONG`, so we should always be adding `c.Config.PingDelay` here, rather than when setting `PingTimeout`.

A value of `300s` for `PingDelay` and a value of `15s` for `PingTimeout` should be a fully supported configuration. I.e. send a `PING` every 5 minutes, and after we send a `PING`, if 15 seconds is reached, error.

The only thing I think should be needed is to swap out the original `60*time.Second` with `PingTimeout`, and then enforce a default of `60s` if `PingTimeout` is `0`.